util/conan-package/.gitignore

```diff
@@ -15,4 +15,5 @@ test_package/build
 conanfile.pyc
 conaninfo.txt
 conanbuildinfo.txt
-conanbuildinfo.cmake
\ No newline at end of file
+conanbuildinfo.cmake
+!FindGLM.cmake
```

Comment:
Why do you need to explicitly include FindGLM? Does it match some other ignore pattern?